pubsub/src/main/java/com/github/twitch4j/pubsub/domain/PredictionEvent.java

```diff
@@ -0,0 +1,31 @@
+package com.github.twitch4j.pubsub.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Setter;
+
+import java.time.Instant;
+import java.util.List;
+
+@Data
+@Setter(AccessLevel.NONE)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PredictionEvent {
+    private String id;
+    private String channelId;
+    private Instant createdAt;
+    private PredictionTrigger createdBy;
+    private Instant endedAt;
+    private PredictionTrigger endedBy;
+    private Instant lockedAt;
+    private PredictionTrigger lockedBy;
+    private List<PredictionOutcome> outcomes;
+    private Integer predictionWindowSeconds;
+    private String status; // e.g. ACTIVE or LOCKED or RESOLVE_PENDING or RESOLVED
```

Comment:
Can we use a enum here as we only have 4 possible values?

Reply:
I think there is at least one more state (e.g. if the prediction was cancelled before being resolved) - I can figure out what `status` is in this scenario but I was afraid of more possible states as this topic is undocumented

Reply:
Then lets keep it as comment on the field as it is currently 👍

Reply:
Should we put this info in a javadoc comment to make it more visible or not bother due to the undocumented nature?

Reply:
Good idea, lets put all currently known states into the javadoc comment - maybe someone discovers the states there before asking what values are possible in discord

Reply:
> javadoc on the field will now be copied to generated getters and setters. Normally, all text is copied, and `@return` is moved to the getter, whilst `@param` lines are moved to the setter. Moved means: Deleted from the field's javadoc. It is also possible to define unique text for each getter/setter. To do that, you create a 'section' named `GETTER` and/or `SETTER`. A section is a line in your javadoc containing 2 or more dashes, then the text 'GETTER' or 'SETTER', followed by 2 or more dashes, and nothing else on the line. If you use sections, `@return` and `@param` stripping for that section is no longer done (move the `@return` or `@param` line into the section).